src/BuildScriptGenerator/DefaultPlatformsInformationProvider.cs

```diff
@@ -52,26 +51,34 @@ public IEnumerable<PlatformDetectorResult> DetectPlatforms(RepositoryContext con
                 }
 
                 var detectionResult = platform.Detect(context);
+
                 if (detectionResult != null)
                 {
-                    detectionResults.Add(detectionResult);
+                    var toolsInPath = platform.GetToolsToBeSetInPath(context, detectionResult);
+
+                    platformInfos.Add(new PlatformInfo
```

Comment:
It looks like the platformInfo has a detectionResult and a dictionary of its content. wondering if we can simplify this if we only want to have them printed out.

Reply:
Let us discuss this in today's meeting.